epitator/spacy_annotator.py

```diff
@@ -3,69 +3,79 @@
 from __future__ import absolute_import
 from .annotator import Annotator, AnnoSpan, AnnoTier
 import re
-from .spacy_nlp import spacy_nlp
+from .spacy_nlp import spacy_nlp, sent_nlp
 
 
 class TokenSpan(AnnoSpan):
     __slots__ = ['token']
 
-    def __init__(self, token, doc):
+    def __init__(self, token, doc, offset=0):
         super(TokenSpan, self).__init__(
-            token.idx,
-            token.idx + len(token),
+            token.idx + offset,
+            token.idx + len(token) + offset,
             doc)
         self.token = token
 
 
 class SentSpan(AnnoSpan):
-    __slots__ = ['span']
+    #__slots__ = ['span']
 
     def __init__(self, span, doc):
         super(SentSpan, self).__init__(
             span.start_char,
             span.end_char,
             doc)
-        self.span = span
+        #self.span = span
```

Comment:
Does this mean that a `SentSpan` will be instantiated without a `.span` property?